web/modules/custom/dbcdk_community/src/Form/FlagsMarkReadForm.php

```diff
@@ -0,0 +1,118 @@
+<?php
+/**
+ * @file
+ * Class definition for FlaggedContentMarkAllFlagsAsReadForm.
+ */
+
+namespace Drupal\dbcdk_community\Form;
+
+
+use DBCDK\CommunityServices\Api\FlagApi;
+use DBCDK\CommunityServices\ApiException;
+use Drupal\Core\Form\FormBase;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\dbcdk_community\Content\FlaggableContent;
+use Psr\Log\LoggerInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Form for marking a list of flags.
+ */
+class FlagsMarkReadForm extends FormBase {
+
+  /**
+   * The flag API to use when marking flags as read.
+   *
+   * @var \DBCDK\CommunityServices\Api\FlagApi
+   */
+  protected $flagApi;
+
+  /**
+   * The logger to use.
+   *
+   * @var LoggerInterface
+   */
+  protected $logger;
+
+  /**
+   * FlagsMarkReadForm constructor.
+   *
+   * @param LoggerInterface $logger
+   *   The logger to use.
+   * @param FlagApi $flag_api
```

Comment:
Heh. As far as I remember it:

If the fully qualified class name (FQCN) is already declared in a use statement and in use for non-comment code you do not have to use the FQCN in comments.

If the class name is only used in comments you must use the FQCN there.